common/workunit/workunit.hpp

```diff
@@ -378,6 +378,29 @@ interface IConstWUAssociatedFileIterator : extends IScmIterator
 };
 
 
+interface IConstWUFieldUsage : extends IInterface // Defines a file (dataset or index) that contains used fields from queries
+{
+    virtual IStringVal & getName(IStringVal & ret) const = 0;
```

Comment:
minor:  For future reference, there has been a move away from functions like this to functions:

    virtual const char * queryName() const = 0;

because it can be implemented more efficiently.  

E.g.,     virtual const char *queryWuid() const = 0; in IConstWorkUnitInfo